crates/typst-library/src/meta/bibliography.rs

```diff
@@ -89,10 +89,16 @@ pub struct BibliographyElem {
     /// The bibliography style.
     #[default(BibliographyStyle::Ieee)]
     pub style: BibliographyStyle,
+
+    /// The loaded bibliography.
+    #[not_hash]
```

Comment:
`#[not_hash]` seems to be only used here. Is it used because `hayagriva::Entry` isn't `Hash`? Not hashing might lead to incremental miscompilations so I don't think `not_hash` should exist. Anyway, rebasing this PR to typst:main should remove the need for it because there the bibliography is also synthesized and I have written an appropriate hayagriva wrapper that deals with hashing (by hashing the original file contents instead).

Reply:
At least it didn't use to be hash. But I'll rebase anyway and figure out what's what ;)

Reply:
The new one is hash 🎉